### ISSUE TITLE
Using only two pushbuttons with new auto tare feature?

### ISSUE THREAD

**Issue by msageryd** [mlsorensen/apollo#8]
Thank you for this writeup!
This is relaly cool. I hadn't decided between the Mini and Micra, but your work certainly makes me lean towards the Micra.

I'm currently looking for some nice buttons for this project, and I was wondering if you would you suggest to not implement the tare button now when you have auto tare in software? Or are there other use for the tare button which is not covered by auto tare?

**Comment by mlsorensen** [comment]
Yes that's a good point. I still use the tare button when I weigh the beans, and auto tare for the shot of course - but there's still the tare on the scale itself.  I actually considered dropping the button for extra profiles because I found I don't really use it.

Might be worth doing a 2.0 with a slimmed down package and interface. I know there are other projects out there as well now that are more simple and work with multiple machines.

**Comment by mlsorensen** [comment]
This project is being archived in favor of https://github.com/mlsorensen/Apollo2. I'm sure if it's working well for you as is, it will continue to do so!.
